lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -144,12 +144,13 @@ def get(path, body = '', options = {})
 
             private
 
-            def faraday_connection
+            def faraday_connection(params = {})
```

Comment:
Since we're in the place where we're creating the Faraday connection, I would probably just call this variable `options` to match with the naming of `faraday_options`.  Also, I've seen a general pattern of using `options` as the hash of optional arguments at the end of a Ruby function.  When I see `params` I think of Rails' controller parameters (that may just be my preference, but I *think* it's a trend)